extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/groovy/org/eclipse/smarthome/binding/hue/test/HueLightHandlerOSGiTest.groovy

```diff
@@ -334,13 +335,38 @@ class HueLightHandlerOSGiTest extends OSGiTest {
         def expectedReply ='{"on" : false}'
         assertSendCommandForBrightness(OnOffType.OFF, currentState, expectedReply)
     }
-
+    
     @Test
     void 'assert command for brightness channel: on'() {
         def currentState = new HueLightState()
         def expectedReply ='{"on" : true}'
         assertSendCommandForBrightness(OnOffType.ON, currentState, expectedReply)
     }
+    
+    @Test
+    void 'assert command for alert channel'() {
+        def currentState = new HueLightState().alert('NONE')
+        def expectedReply ='{"alert" : "none"}'
+        assertSendCommandForAlert(new StringType("NONE"), currentState, expectedReply)
+        currentState.alert("SELECT")
+        expectedReply ='{"alert" : "select"}'
+        assertSendCommandForAlert(new StringType("SELECT"), currentState, expectedReply)
```

Comment:
Wouldn't it make sense to set the currentState to something else than 'SELECT', e.g. 'NONE', and then verify that the reply is "select" when sending the "SELECT" command? This is also valid for the other tests.